src/Core/Settings.cs

```diff
@@ -391,6 +391,12 @@ public class UserUIData : AutoConfiguration
             [ConfigComment("If enabled, shifting to next/previous image (eg with arrow keys) in history or batch view,\ncycles at the ends (jumps from the start to the end or vice versa).\nIf disabled, shifting will simply stop at the ends.\nIf 'only arrow keys', cycling happens when you press the arrow keys, but not other actions (eg deleting an image will not cycle).")]
             [ManualSettingsOptions(Vals = ["true", "false", "only_arrows"], ManualNames = ["Enabled", "Disabled", "Only Arrow Keys"])]
             public string ImageShiftingCycles = "true";
+
+            [ConfigComment("When a model with a linked preset is selected, should the preset parameters be applied immediately? By default, the preset is just selected for review.")]
+            public bool AutoApplyModelPresets = false;
```

Comment:
It'd probably make more sense for this to be a setting on each Preset rather than a global user setting. Some presets should autoapply, others shouldn't. The functionality would probably be beneficial for all presets rather than specific to model. This should aaalso probably be a separate PR.